File: app/lib/core/carreras.dart
/// RF-01: los 3 nombres de carrera son el texto literal del ERS (no la forma
/// abreviada que trae diseno-tecnico.md §2 — ver
/// backend/src/common/carreras.ts, la misma lista del lado del servidor).
/// Compartida entre el registro de alumno (T-010) y los filtros del panel
/// docente (T-053, RF-30) para no duplicarla en dos archivos.
const carreras = [
  'Ingeniería en Agroalimentos',
  'Ingeniería en Desarrollo de Software',
  'Licenciatura en MiPymes',
];

File: backend/src/common/carreras.ts
// RF-01/RF-37. Texto literal del ERS (no la forma abreviada que trae
// diseno-tecnico.md §2, que transcribió mal estos 3 valores). Compartido
// entre RegistroAlumnoDto (T-011) y los filtros de progreso (T-052, RF-30)
// para que ambos validen contra exactamente el mismo enum, sin duplicarlo.
export const CARRERAS = [
  'Ingeniería en Agroalimentos',
  'Ingeniería en Desarrollo de Software',
  'Licenciatura en MiPymes',
] as const;

export type Carrera = (typeof CARRERAS)[number];
